Changes

```diff
@@ -1,6 +1,59 @@
 OCaml 4.07
 ----------
 
+Some highlights of this release are:
+
+- The way the standard library modules are organized internally has
+  changed (GPR#1010):
+  1. the List module (for example) is now named Stdlib__list
+  2. a new Stdlib module contains a series of aliases
+  such as `module List = Stdlib__list`
+  3. the Stdlib module is implicitly opened when type-checking OCaml
+  programs (as Pervasives previously was), so that Stdlib.List can be
+  accessed as just List, as before.
+
+  This should be invisible to most users, although it is possible that
+  some tools show the Stdlib. or Stdlib__ prefixes in messages. (You
+  might want to report these situations as usability bugs.) The change
+  prevents standard library modules from conflicting with end-user
+  filenames (please avoid stdlib.ml and the stdlib__ prefix); we may
+  introduce new standard library modules in the future with less fear
+  of breaking user code. In particular, Float (GPR#1638) and Seq
+  (GPR#1002) modules have now been added to the standard library.
```

Comment:
I was wondering if the transition of `Bigarray` to the standard library should also be mentioned here.

Reply:
It's rather independent -- the move is not justified by the Stdlib prefixing. Would you remind me what is the potential impact for the user of the Bigarray change? (Does explicitly linking with `bigarray.cm[x]a` still work?) How would they notice the difference?

Reply:
I completely agree that the change is quite independent. However, it is also a new module in the standard library, and it seems possible to misread the previous paragraph and conclude that `Seq`and `Float` are the only two new modules. (To clarify my position, I am not insisting that the Bigarray move should be mentionned, just detailing why I think it could contingently make sense to do so.)

Concerning the changes themselves, they should be completely transparent to current bigarray's users since the extended Bigarray module overrides the standard library's one by default. However, it also means that it is quite easy to miss the fact that it is now possible to use Bigarray without depending on Unix and `bigarray.cm[x]a` by using `Stdlib.Bigarray` rather than `Bigarray`.

Reply:
How about just having a distinct change line noting Bigarray is now in the standard library so people can't confuse the two?

Reply:
This is in the Changes below (it's the fourth entry, second in "Standard library" section), do we believe that this change in particular should be feature in the "Highlights of the release"?